examples/package.yaml

```diff
@@ -0,0 +1,3 @@
+dependencies:
+  newton_raphson:
```

Comment:
Kebab?

Reply:
I'm leaving the names that are generated by 'pkg toit'. (This is the name given by the package).

The package manager needs to change to make this nicer.